content/english/secret-messages/activity-5.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Activity 5 - ASCII Code"
+date: 2019-07-25T13:24:17-07:00
+weight: 6
+draft: true
+---
+
+That's great!
+
+In this activity we will encrypt (make secret) the name of the recipient.
+
+Let's use the **ASCII Code**
+
+{{% notice tip %}}
+
+### ASCII Code
+
+The ASCII code shows that each character has a numeric value. For example, the character `A` is the number 65, `B` is 66, `C` is 67, etc.
+
+The following list shows the value of the most commonly used characters:
+
+![List of ASCII Codes](../media/ascii-english.png)
+
+Since each character has a numeric value, we can use this to add or subtract values to these letters, and thus change the order of them in a certain way. It allows us to convert characters (`char`) into numbers (`int`) and vice versa.
+
+Example
+```
+char letter = 'A';
+int letter_value = letter;
+
+cout << letter << " = " << letter_value << endl;
+```
+This will print:
+```
+A = 65
+```
+
+In this example, we save the character in a variable type **char**. Then, in an **int** type variable we save the char type variabale, but since the second variable saves numbers, it will convert the character into its numerical value according to the ASCII code.A
```

Comment:
```suggestion
In this example, we save the character in a variable type **char**. Then, in an **int** type variable we save the char type variable, but since the second variable saves numbers, it will convert the character into its numerical value according to the ASCII code.
```